docs/Validation.md

```diff
@@ -513,3 +513,44 @@ Now, update the xsi:schemaLocation with bad location hint
 ```
 
 In `always` you will have error, in `onValidSchema` you will have none error.
+
+## xml.validation.filters
+
+An XML validation filter gives you the capability to defines some validation rule for a given pattern file. For instance if you wish to disable validation for all files which have `*.myxml` file extension but not for all other XML files, you must declare this filter in the settings.json:
+
+```json
+"xml.validation.filters": [
+   // Declaration of validation filter to disable validation for all *.myxml files.
+   {
+    "pattern": "**.myxml",
+    "enabled": false
+  }
+]
+```
+
+You can use other validation settings (enabled, noGrammar, etc). For instance if you wish to remove the warning no grammar for all files which have `*.myxml` file extension but not for all other XML files, you must declare this filter in the settings.json:
```

Comment:
... if you wish to remote the *"no grammar" warning* ...

Also remove "but not for all other XML files" here, since this is implied.

Reply:
fixed